operator/controllers/btpoperator_controller.go

```diff
@@ -52,25 +52,30 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+var (
```

Comment:
Just thinking... what is the reason behind making them global environment variables and not part of the  BtpOperatorReconciler struct?

Reply:
so they can be easily flag parsed
https://github.com/kyma-project/btp-manager/blob/fdc8bcb01143a7590c1e19f5f3f2f52e83e5453d/operator/main.go#L59-L72

can drop the CLI support and move them to the controller if that is more desirable.

Reply:
no, that's fine. We can adjust later if needed.